src/atoms/Button/Button.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+
+const Pulse = keyframes`
```

Comment:
When I reviewed the actual app you got me thinking you had used MaterialUI, but this is where that 'pulse' is coming from. Awesome!.